service/dynamodb/dynamodbattribute/encode.go

```diff
@@ -194,6 +194,13 @@ type MarshalOptions struct {
 	// Note that values provided with a custom TagKey must also be supported
 	// by the (un)marshalers in this package.
 	TagKey string
+
+	// EmptyCollections modifies how structures, maps, and slices are (un)marshalled.
+	// When set to true empty collection values will be preserved as their respective
+	// empty DynamoDB AttributeValue type when set to true.
+	//
+	// Disabled by default.
+	EmptyCollections bool
```

Comment:
Should this be named something slightly more specific like `EnableEmptyCollections`? Might help discoverability and expectations.